### PR TITLE
Add flat option to AggregateRESTService

### LINKED ISSUE #397: extend the 'flat' option to Aggregate REST service

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
Fix #397

### CODE REVIEW

File: arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java [hunk 1]
```diff
@@ -158,7 +168,7 @@ public Response aggregate(
         request.basicRequest = aggregationsRequest;
         request.headerRequest = aggregationsRequestHeader;
 
-        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference);
+        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference, (flat != null && flat));
```
Comment (mbarbet) [inline arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java]: you could use Boolean.TRUE.equals(flat)

File: arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java [hunk 2]
```diff
@@ -222,13 +233,13 @@ public Response aggregatePost(
         request.basicRequest = aggregationsRequest;
         request.headerRequest = aggregationsRequestHeader;
 
-        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference);
+        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference, (aggregationsRequest.form != null && aggregationsRequest.form.flat != null && aggregationsRequest.form.flat));
         aggregationResponse.totalTime = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startArlasTime);
```
Comment (mbarbet) [inline arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java]: you could use Boolean.TRUE.equals(aggregationsRequest.form.flat)

File: arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java (hunk 1 above)
Comment (MohamedHamouGisaia) [reply arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java]: Done

File: arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java (hunk 2 above)
Comment (MohamedHamouGisaia) [reply arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java]: Done

Comment (mbarbet) [reply arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java]: 👍

**Pull request merged**